test/e2e/framework/util.go

```diff
@@ -163,6 +163,9 @@ const (
 
 	// Number of times we want to retry Updates in case of conflict
 	UpdateRetries = 5
+
+	// number of objects that gc can delete in a second
```

Comment:
Can you please add a comment with what @caesarxuchao mostly wrote (that we have two operations per delete) (to show that this 10 is not a random number).